src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrintDocument.Windows.cs

```diff
@@ -14,15 +14,15 @@ public class PrintDocument : Component
     {
         private string _documentName = "document";
 
-        private PrintEventHandler _beginPrintHandler;
-        private PrintEventHandler _endPrintHandler;
-        private PrintPageEventHandler _printPageHandler;
-        private QueryPageSettingsEventHandler _queryHandler;
+        private PrintEventHandler? _beginPrintHandler;
+        private PrintEventHandler? _endPrintHandler;
+        private PrintPageEventHandler? _printPageHandler;
+        private QueryPageSettingsEventHandler? _queryHandler;
 
         private PrinterSettings _printerSettings = new PrinterSettings();
         private PageSettings _defaultPageSettings;
 
-        private PrintController _printController;
+        private PrintController? _printController;
 
         private bool _originAtMargins;
         private bool _userSetPageSettings;
```

Comment:
Below on row 151-196 event accessors `BeginPrint, EndPrint, PrintPage, QueryPageSettings` should be nullable

Reply:
Why do they need to be nullable?

Reply:
I am not sure about that, but that is what we were doing before for event accessors CC @stephentoub